packages/web-forms/CHANGELOG.md

```diff
@@ -4,7 +4,7 @@
 
 ### Minor Changes
 
-- 8261eee: Support for uploading images
+- [8261eee](https://github.com/getodk/web-forms/pull/363): Support for uploading images
```

Comment:
@lognaturel Linking a PR is something we don't do in the changelog.

Sadiq mentioned that `8261eee` is pointing to a commit adding the image upload changelogs, but that's not super useful. Many commits together represent the work adding support for the image upload (engine and web forms). So, linking the PR would be more useful this time. 

I checked some other commit references in these changelogs, and they seem okay. These seem to link better when squash-merge small targeted PRs.

Reply:
I'd like to hear more about what Sadiq is looking for and address that as its own piece of work if that's ok with you. Agree the commit hashes are not always all that helpful and it'd be great to come up with a process/structure that makes the changelog as helpful as it can be. I am guessing `changeset` has options that would help with that.

Reply:
That's okay. I will revert this commit and merge it. 

Thanks for the quick review!